proto/squad/farming/v1beta1/query.proto

```diff
@@ -121,6 +122,30 @@ key:
 };
 }
 
+rpc Position(QueryPositionRequest) returns (QueryPositionResponse) {
```

Comment:
We need to update `docs/api/farming.md` and `docs/cli/farming.md` documents for this newly added query.

Reply:
You're right. I'll fix this.